packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
```

Comment:
A new version was released, and this section is now released. You'll have to rebase and move the new entries to the new "Unreleased" section.